sprint 8/tema 1.js

```diff
@@ -0,0 +1,10 @@
+const numbers = [];
+let n = 10;
+
+    for (let i = 0; i <= n; i++) {
```

Comment:
Nu e nevoie de acest spatiu in plus aici. 
Se poate de identat cu un tab (4 spatii) mai la stanga toate liniile 4-8